corehq/apps/app_execution/static/app_execution/js/workflow_charts.js

```diff
@@ -3,7 +3,8 @@ hqDefine("app_execution/js/workflow_charts", [
     'jquery',
     'moment/moment',
     'd3/d3.min',
-    'nvd3/nv.d3.latest.min',  // version 1.1.10 has a bug that affects line charts with multiple series
+    'nvd3-1.8.6',  // version 1.1.10 has a bug that affects line charts with multiple series
```

Comment:
Version numbers should normally only be in package.json so we don't have to change other files when upgrading. Is it possible to use some alias/name that does not include the version number?

Reply:
That makes sense. I added an alias in the webpack config so that application code doesn't need the version number, does that work? 429a2caf170